test/test_testing.py

```diff
@@ -311,12 +140,9 @@ def test_isclose_comparetensors_float(self, device, dtype):
 
         self._isclose_helper(tests, device, dtype, True)
 
-        self._comparetensors_helper(tests, device, dtype, True)
-
-    # Note: compareTensor will compare the real and imaginary parts of a
-    # complex tensors separately, unlike isclose.
+    @unittest.skipIf(IS_SANDCASTLE, "Skipping because doesn't work on sandcastle")
```

Comment:
This test was already skipped by sandcastle through internal means. Due to the rename of the test the old skip is no longer in place and cannot be activated preemptively. Thus, we now publicly skip it.